neuromation/api/storage.py

```diff
@@ -121,7 +121,10 @@ def _uri_to_path(self, uri: URL, *, normalized: bool = False) -> str:
 
     def _set_time_diff(self, request_time: float, resp: aiohttp.ClientResponse) -> None:
         response_time = time.time()
-        server_timetuple = parsedate(resp.headers.get("Date"))
```

Comment:
See also https://github.com/python/typeshed/issues/4713.

Although the behavior of `parsedate(None)` is not explicitly documented and therefore your code is more future proof.

Reply:
this was my thought